suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -58,11 +65,16 @@ export const TransactionListItem = memo(({ transaction }: AccountTransactionList
     const transactionAmount = formatNetworkAmount(transaction.amount, transaction.symbol);
     const fiatAmount = toFiatCurrency(transactionAmount, fiatCurrency.label, transaction.rates);
 
-    const getTransactionTimestamp = () => {
+    const getTransactionDatetime = () => {
```

Comment:
rename to `formatTransactionDate` + add comment that it should be replaced with Datetime formatter once it will be ready

Reply:
fixed in [30b5f1a](https://github.com/trezor/trezor-suite/pull/7392/commits/30b5f1a27811ed560da7091928ce739d439d0ca9)